spec.md

```diff
@@ -52,20 +52,19 @@ A digital signature scheme is a fundamental cryptographic primitive that is used
 
 However traditional digital signatures are limited to fixed mode of signing and verifying, that is the entire payload that was signed by a signer must be known by the verifier in-order to validate the digital signature.
 
-This document describes the BBS signature scheme. The scheme features many important properties:
+This document describes the BBS signature scheme. (FIXME: THE BBS signature scheme? Or A BBS signature schemes? Also: BBS citation needed/informative reference).
+The scheme feature important properties that allow the scheme to be used in applications where privacy and data minimization techniques are desired and/or required:
 
-1. The signature is over a group of Pedersen commitments--signatures can be created blinded or un-blinded.
+1. Signatures can be created blinded or un-blinded.
    
-2. The signature is encoded as a single group element and two field elements.
+2. Signatures are encoded as a single group element and two field elements. (FIXME: Why is this important? Succinctness? Unclear)
 
-3. Verification requires 2 pairing operations.
+3. Verification requires 2 pairing operations. (FIXME: Only two? Why is this important? It's not as good as one or no pairing!)
```

Comment:
Same here, not sure what the suggested change would be.

Reply:
Well let me rephrase it using the words from the (original) text:
How is "Verification requires 2 pairing operations." a "[property that] allow[s] the scheme to be used in applications where privacy and data minimization techniques are desired and/or required."?
That is what the text currently says, and it is unclear (to me) where the connection is.

Reply:
+1 as above I think this should be removed also